eip-191-signing/browser/src/index.js

```diff
@@ -0,0 +1,142 @@
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LitNetwork } from "@lit-protocol/constants";
+import {
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+} from "@lit-protocol/auth-helpers";
+import { disconnectWeb3 } from "@lit-protocol/auth-browser";
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import * as ethers from "ethers";
+
+import { litActionCode } from "./litAction";
+
+document.addEventListener("DOMContentLoaded", () => {
+  document.getElementById("myButton").addEventListener("click", buttonClick);
+});
+
+async function buttonClick() {
+  try {
+    console.log("Clicked");
+
+    const provider = new ethers.providers.Web3Provider(window.ethereum);
+    await provider.send("eth_requestAccounts", []);
+    const ethersSigner = provider.getSigner();
+    console.log("Connected account:", await ethersSigner.getAddress());
+
+    const litNodeClient = await getLitNodeClient();
+
+    const sessionSigs = await getSessionSigs(litNodeClient, ethersSigner);
+    console.log("Got Session Signatures!");
+
+    const litActionSignatures = await litNodeClient.executeJs({
+      sessionSigs,
+      code: litActionCode,
+      jsParams: {
+        dataToSign: ethers.utils.arrayify(
+          ethers.utils.keccak256([1, 2, 3, 4, 5])
+        ),
+        publicKey: await getPkpPublicKey(ethersSigner),
+        sigName: "sig",
+      },
+    });
+    console.log("litActionSignatures: ", litActionSignatures);
+
+    verifySignature(litActionSignatures.signatures.sig);
+  } catch (error) {
+    console.error(error);
+  } finally {
+    disconnectWeb3();
+  }
+}
+
+async function getLitNodeClient() {
+  const litNodeClient = new LitNodeClient({
+    litNetwork: LitNetwork.Cayenne,
+  });
+
+  console.log("Connecting litNodeClient to network...");
+  await litNodeClient.connect();
+
+  console.log("litNodeClient connected!");
+  return litNodeClient;
+}
+
+async function getPkpPublicKey(ethersSigner) {
+  if (
+    process.env.PKP_PUBLIC_KEY !== undefined &&
+    process.env.PKP_PUBLIC_KEY !== ""
+  )
```

Comment:
can we avoid having process envs inside a function, and put it at the top of the file instead so we know what to configure when we look at this file

Reply:
That's the intention of the [.env.example](https://github.com/LIT-Protocol/developer-guides-code/blob/wyatt/eip-191/eip-191-signing/nodejs/.env.example) file though. I'd prefer to avoid global variables when they're only used in a single function. I also plan to be more in depth about what envs are required in the `README` as done [here](https://github.com/LIT-Protocol/developer-guides-code/blob/wyatt/conditional-signing/conditional-signing/nodejs/README.md#setting-up-the-env-file)

Reply:
yea, we can still have `.env`, but we need process envs at the top of the file. It's a common practice for other devs. to understand what configurations are needed for this file/module, which you might not necessary need all the variables declared in the `.env` file

Reply:
Yea that makes sense, will update!

Reply:
@Ansonhkg [Remove process.env reads](https://github.com/LIT-Protocol/developer-guides-code/pull/5/commits/e7c03e8dd31cc9b18bdda3b7b6ec9288a2c6e0ba)!